src/core.js

```diff
@@ -215,6 +215,20 @@ define([
             (hotkeys[key] || _.identity).call(_this, e);
         });
 
+        $(document).on('click', '.jstree-hover', function(e) {
+            e.preventDefault();
+            _this.jstree("hover_node", $(this).data("ufid"));
+            var $node = $(".jstree-hovered");
+            if ($node.length) {
+                var $scrollable = $node.closest(".fd-scrollable");
+                $scrollable.scrollTop($node.position().top - $scrollable.position().top);
+            }
+            if (window.analytics) {
```

Comment:
not immediately actionable, but it feels like these analytics calls are being littered across the code. I wonder if its worth it to have it in a plugin or as an `eventuality` that we can call?